### DISCUSSION TITLE
Managed hosting option

### DISCUSSION THREAD

**Discussion by m3nu** [ArchiveBox/ArchiveBox#973] [General]
Over at [PikaPods.com](https://www.pikapods.com/), we are on a mission to make great open source apps, like ArchiveBox more accessible and simpler to get started with.

We already offer ArchiveBox in our “app store” and reception has been pretty good from the start.

So I wanted to check if there is interest in adding PikaPods as additional deployment option in the documentation. This would help an even wider audience to benefit from the project.

Looking forward to hearing the team’s opinion on this. Thanks for the consideration!

**Comment by pirate** [comment]
Go for it, just PR to add it to the paid hosting list in the README and or the Wiki docs.

**Reply by m3nu** [reply]
Great! Already added a PR: https://github.com/ArchiveBox/ArchiveBox/pull/974

Wiki only has [2 entries](https://github.com/ArchiveBox/ArchiveBox/wiki/Web-Archiving-Community#managed-archivebox-hosting). Could make sense to merge it with the README section to have it all in one place? Want a PR for that?
